README.md

```diff
@@ -13,21 +18,38 @@ Install both backend and frontend dependencies:
 ```bash
 make install
 ```
+## How to setup
 
-## How to run locally
+First, ensure you have Node.js installed. It is recommended to use Node Version Manager (nvm) to manage Node.js versions:
+
+1. Install nvm:
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+```
+
+2. Install Node.js using nvm:
+```bash
+nvm install 22.12.0
```

Comment:
The readme suggests you were using node version 22, but the backend docker is using version 18 alpine image and the front end is using version 16.  Can you align on a single version and use it ion docker too.

Reply:
Using version 18.19.0. Dockerfiles use 18 👍